recipes/gistim/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "gistim" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gistim-{{ version }}.tar.gz
+  sha256: fdf105534fd38121b8cbec7066f0626d952370a854f7c40845030c0e3315f7b0
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools_scm
+  run:
+    - affine
+    - geopandas
+    - matplotlib
```

See https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib
```suggestion
    - matplotlib-base
```